src/Teams/Startup.cs

```diff
@@ -31,6 +33,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddRazorPages();
             services.AddScoped<IManageTeamsMembersService, ManageTeamsMembersService>();
             services.AddScoped<IManageTeamsService, ManageTeamsService>();
```

Comment:
Was my suggestion to use scoped, but yep, better to use Transient
Please change for IManageTeamsMembersService, IManageTeamsService to transient